website/docs/d/client_end_point.html.markdown

```diff
@@ -11,39 +11,49 @@ description: |-
 
 Data source for ACI Client End Point
 
+## API Information ##
+
+* `Class` - fvCEp
+* `Distinguished Name` - uni/tn-{tenant_name}/ctx-{ctx_name}/cep-{client_end_point_name}
+* `Distinguished Name` - uni/tn-{tenant_name}/ap-{ap_name}/epg-{epg_name}/cep-{client_end_point_name}
+* `Distinguished Name` - uni/tn-{tenant_name}/l2out-{l2out_name}/instP-{instance_profile_name}/cep-{client_end_point_name}
```

Comment:
Does this terraform module accept all the 3 Dns?

Reply:
As per the code, this module should accept all 3 Dns. But I couldn’t test it.